examples/tanstack/src/components/SignIn.tsx

```
@@ -193,261 +192,271 @@ export const SignIn = () => {
+                Sign in with Google
+              </Button>
+            </form>
+          </CardContent>
+          <CardFooter>
+            <div className="flex justify-center w-full border-t py-4">
+              <p className="text-center text-xs text-neutral-500">
+                Powered by{' '}
+                <a
+                  href="https://better-auth.com"
+                  className="underline"
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)